episodes/01-introduction.md

```diff
@@ -1,101 +1,94 @@
+### An example: Making a coupled model of a flying aeroplane
+![Computational fluid dynamics solver applied to A380 aircraft. (Image: DLR, CC-BY 3.0) ](../fig/640px-DLR_2007_A380_sim_hires.jpg){alt='Computational fluid dynamics simulation of airflow over an aeroplane wing.'}
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+Imagine that we have two computational models: Model A and Model B.
+Model A simulates how an aeroplane moves in response to forces on its body.
+Model B simulates airflow around objects and calculates the forces generated by that airflow.
+We could pass the shape, speed and rotation of the airplane as input to Model B, allowing it to calculate the forces on the plane.
+In turn, we could then pass the calculated forces back to Model A, allowing it to calculate the effect of the forces on the plane's speed, position and rotation.
+In this way we have _coupled_ the two models together, making a new, more complex computational model. The two original models are _submodels_ of the coupled model.
 
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
+![Models A and B for the plane coupling example](../fig/ep01-plane-model-coupling.png){alt='One box labelled Model A and another box labelled Model B. There is an arrow between A and B showing transfer of plane shape. There is an arrow from B to A showing transfer of resulting airflow stresses.'}
 
-::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: challenge 
 
-## Challenge 1: Can you do it?
-
-What is the output of this command?
-
-```r
-paste("This", "new", "lesson", "looks", "good")
-```
+### Exercise: Breaking it down
+Think of another system you might want to simulate.
+Can you think of how to break it down into two or more computational submodels?
+Most importantly, consider what information would be passed from one model to the other.
 
 :::::::::::::::::::::::: solution 
 
-## Output
- 
-```output
-[1] "This new lesson looks good"
-```
-
-:::::::::::::::::::::::::::::::::
+### Example solution
 
-
-## Challenge 2: how do you nest solutions within challenge blocks?
-
-:::::::::::::::::::::::: solution 
-
-You can add a line with at least three colons and a `solution` tag.
+- Local climate-ecology: One model simulates the expected climate of a region of the Earth under certain vegetation/land use conditions. A separate model simulates the proliferation and behaviour of animals and vegetation in a given region under influence of various climatic conditions.
+- Stent in a blood vessel: A computational fluid dynamics model (simulating the blood flow) through a metal alloy stent. The stresses created by the blood on the stent are passed to a mechanical model of the stent, and used to deform the stent accordingly. The current deformation state of the stent is passed to the blood flow model.
```

Comment:
Do stents get deformed by blood flow?

Reply:
Not really, this was a bit of a lazy example. I have changed it now to be more about in-stent restenosis (still very simplified)